script.js

```diff
@@ -1,4 +1,165 @@
+    <br> Hit submit to play again. <br> <br>For the record, player 1's total is ${player1Sum}, and player 2's total is ${player2Sum}. 
+    <br><br> Leaderboard: <br> 1. Player 2 <br> 2. Player 1 `
+  } else if (player1Number > player2Number && player1Sum == player2Sum) {
+    myOutputValue = `Player 1 scored ${player1Number} and player 2 scored ${player2Number}. Player 1 wins this round. <br>
+    <br> Hit submit to play again. <br> <br>For the record, player 1's total is ${player1Sum}, and player 2's total is ${player2Sum}. 
+    <br><br> Leaderboard: <br> Shoulder to shoulder! `
+  }  else if (player1Number == player2Number && player1Sum > player2Sum) {
+    myOutputValue = `Player 1 scored ${player1Number} and player 2 scored ${player2Number}. It's a draw this round. <br>
+    <br> Hit submit to play again. <br> <br>For the record, player 1's total is ${player1Sum}, and player 2's total is ${player2Sum}.
+    <br><br> Leaderboard: <br> 1. Player 1 <br> 2. Player 2 `
+  } else if (player1Number == player2Number && player1Sum < player2Sum) {
+    myOutputValue = `Player 1 scored ${player1Number} and player 2 scored ${player2Number}. It's a draw this round. <br>
+    <br> Hit submit to play again. <br> <br>For the record, player 1's total is ${player1Sum}, and player 2's total is ${player2Sum}.
+    <br><br> Leaderboard: <br> 1. Player 2 <br> 2. Player 1 `
+  }else if (player1Number == player2Number && player1Sum == player2Sum) {
+    myOutputValue = `Player 1 scored ${player1Number} and player 2 scored ${player2Number}. It's a draw this round. <br>
+    <br> Hit submit to play again. <br> <br>For the record, player 1's total is ${player1Sum}, and player 2's total is ${player2Sum}.
+    <br><br> Leaderboard: <br> Shoulder to shoulder! `
+   } else if (player1Number < player2Number && player1Sum > player2Sum) {
+    myOutputValue = `Player 1 scored ${player1Number} and player 2 scored ${player2Number}. Player 2 wins this round. <br>
+    <br> Hit submit to play again. <br> <br>For the record, player 1's total is ${player1Sum}, and player 2's total is ${player2Sum}. 
+    <br><br> Leaderboard: <br> 1. Player 1 <br> 2. Player 2 `
+  } else if (player1Number < player2Number && player1Sum < player2Sum) {
+    myOutputValue = `Player 1 scored ${player1Number} and player 2 scored ${player2Number}. Player 2 wins this round. <br>
+    <br> Hit submit to play again. <br> <br>For the record, player 1's total is ${player1Sum}, and player 2's total is ${player2Sum}. 
+    <br><br> Leaderboard: <br> 1. Player 2 <br> 2. Player 1 `
+  }  else if (player1Number < player2Number && player1Sum == player2Sum) {
+    myOutputValue = `Player 1 scored ${player1Number} and player 2 scored ${player2Number}. Player 2 wins this round. <br>
+    <br> Hit submit to play again. <br> <br>For the record, player 1's total is ${player1Sum}, and player 2's total is ${player2Sum}. 
+    <br><br> Leaderboard: <br> Shoulder to shoulder!`
```

Comment:
There is a lot of repeated code in the myOutputValue. Remember that we try to follow the DRY principle when coding. D-ont' R-epeat Y-ourself. 

You could consider storing the first part in one variable:
var roundResult = `Player 1 scored ${player1Number} and player 2 scored ${player2Number}.`
This can just be written once above line 92.

We also declare a var overallRecord once above line 92:
var overallRecord = `<br>
    <br> Hit submit to play again. <br> <br>For the record, player 1's total is ${player1Sum}, and player 2's total is ${player2Sum}. 
    <br><br> `


Next we can also declare a var roundWinner = ' '; above line 92, and in each conditional statement we can reassign to `Player 1 wins this round. / Player 2 wins this round. It's a draw this round`

We also declare a var leaderboard = ' '; above line 92 and reassign them in each conditional statement.

Finally, we only write myOutputValue ONCE below line 128 and it would combine all the above variables,
var myOutputValue = `${roundResult} ${roundWinner} ${overallRecord} ${leaderboard}`

This will make your code much cleaner!